classes/tests/critical.php

```diff
@@ -144,10 +144,8 @@ public function _oldClassConstructors($line) {
 		}
 
 		// find the start of PHP class declaration
-		if (strpos($line, 'class') === 0) {
-			if (preg_match('#class (\w+)#', $line, $matches)) {
-				$lastClassName = $matches[1];
-			}
+		if (preg_match('#^(abstract\s+)?class (\w+)#', $line, $matches)) {
```

Comment:
Minor regression, that ^ beginning of line character will not match code that has white space at the beginning.(Poor formatting and such.)